src/gawronski-bartosz/Fsd.Bartek.Cs2/Fsd.Bartek.Cs2.Ex2/Domain/Animals/Cat.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
+
+namespace Fsd.Bartek.Cs2.Ex2.Domain.Animals
+{
+    public class Cat : AnimalVoice
+    {
+        public string VoiceString { get;  }
```

Comment:
może to lepiej w klasie bazowej umieścić? I tak występuje w każdej klasie pochodnej